modin/pandas/io.py

```diff
@@ -145,30 +149,30 @@ def _read_csv_from_file(filepath, npartitions, kwargs={}):
         index_ids = []
         total_bytes = os.path.getsize(filepath)
         chunk_size = max(1, (total_bytes - f.tell()) // npartitions)
+        num_splits = min(len(column_names), RayBlockPartitions._compute_num_partitions())
+
         while f.tell() < total_bytes:
             start = f.tell()
             f.seek(chunk_size, os.SEEK_CUR)
             f.readline()  # Read a whole number of lines
 
-            if f.tell() >= total_bytes:
-                kwargs["skipfooter"] = skipfooter
-                kwargs["skip_footer"] = skip_footer
-
-            partition_id, index_id = _read_csv_with_offset.remote(
-                filepath, start, f.tell(), partition_kwargs_id, prefix_id)
-            partition_ids.append(partition_id)
-            index_ids.append(index_id)
+            partition_id = _read_csv_with_offset_pandas_on_ray._submit(args=(filepath, num_splits, start, f.tell(), partition_kwargs_id, prefix_id), num_return_vals=num_splits + 1)
```

Comment:
Might fail lint

Reply:
Yes, we are just going to run the other PR through yapf before the final review. I haven't been worrying about lint until we have everything ready.